fiaas_skipper/web/__init__.py

```diff
@@ -33,9 +34,9 @@ def create_webapp(deployer, cluster):
     from .nav import nav
     app = Flask(__name__)
     # TODO: These options are like this because we haven't set up TLS
-    csp = {'default-src': SELF, 'script-src': [SELF], 'style-src': [SELF]}
+    csp = {'default-src': SELF, 'script-src': [SELF, UNSAFE_INLINE], 'style-src': [SELF, UNSAFE_INLINE]}
```

Comment:
If you have removed all the inline JS, do we still need the `unsafe-inline` CSP?

Reply:
Otherwise it doesn't allow the externalized js files coming from self (served from /static) .